### PR TITLE
[DOCS-419] Adding sendHubCommand() doc

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Initial draft of sendHubCommand() method documentation. This will go in API reference doc. 
@workingmonk please review. If you can add any more detail, in specific an example code fragment for sending a List of HubAction commands, it'll be very helpful. Thanks!

### CODE REVIEW

**Comment by rajkaramchedu** [comment]
@workingmonk Thanks for your great help! Amended the commit with revisions. Wait for your blessing, thanks!

File: ref-docs/smartapp-ref.rst [hunk 1]
```diff
@@ -1676,6 +1676,49 @@ Creates and sends an event constructed from the specified properties. If a devic
...
+    ``List<HubAction> actions`` - A list of HubAction objects 
+
+
+    ``delay`` - An integer number representing milliseconds. This is the delay between commands when a list of HubAction objects are sent using ``List<HubAction> actions`` parameter. The default value of delay is 1000.
+
+**Returns:**
+    void
+
+**Example:**
+    During the discovery phase of a LAN-connected device, after an SSDP search response is received from the Hub, the following discovery command can be sent to the Hub.
```
Comment (workingmonk) [inline ref-docs/smartapp-ref.rst]: "During the discovery phase of a LAN-connected device, the following discovery command can be sent to the Hub."

the part "after an SSDP search response is received from the Hub" makes no sense here

**Comment by workingmonk** [comment]
looks good except for my one comment

**Comment by rajkaramchedu** [comment]
@workingmonk thanks! Made the changes and this is ready to merge.

**Comment by rajkaramchedu** [comment]
@dbradmit this PR is to explicitly address the sendHubCommand() user issue. See if this suffices for now. Documenting HubAction() is my next focus.

**Pull request merged**